misc/docker/full/Dockerfile

```diff
@@ -54,11 +57,24 @@ RUN mkdir -p /opt \
 	# Android NDK
 	&& curl http://dl.google.com/android/repository/android-ndk-r11c-linux-x86_64.zip -o android-ndk.zip \
 	&& unzip -q android-ndk.zip \
+	&& chmod -R a+X /opt \
 	&& ln -s android-ndk-r11c android-ndk \
 	&& rm android-ndk.zip \
-	&& printf "PATH=$PATH:$ANDROID_HOME/tools:$ANDROID_HOME/platform-tools:$ANDROID_NDK\nexport PATH\n" >> "/etc/profile.d/android.sh"
+	&& printf "PATH=$PATH:$ANDROID_HOME/tools:$ANDROID_HOME/platform-tools:$ANDROID_NDK\nexport PATH\n" >> "/etc/profile.d/android.sh" \
+	&& echo y | /opt/android-sdk-linux/android update sdk -u -t android-10,android-15,android-16,android-19,android-21
```

Comment:
Why keep this line separated from the other `android-sdk-linux/tools/android update` a few lines up?